### PR TITLE
Update Transfigure README

### PR DESCRIPTION
**Pull request by chases2**
Encourages renaming the Prow Job, to prevent several Prow Jobs with the
same name.

/cc @michelle192837 @fejta @Katharine

### CODE REVIEW

File: testgrid/cmd/transfigure/README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ To have your instance of Prow push to TestGrid automatically, add a [Prow Job](/
 similar to the one below:
 ```yaml
 postsubmits:
-  YOUR-ORG/YOUR-REPO: # Replace this
-  - name: post-test-infra-upload-testgrid-config
-    cluster: test-infra-trusted # You may need to specify a cluster, depending on your Prow config
-    run_if_changed: '^(dir/to/your/prowjobs/.*\.yaml)|(dir/to/your/testgrid/default\.yaml)$' # Replace this
+  YOUR-ORG/YOUR-REPO: # TODO: Replace this
+  - name: post-[repo-name]-upload-testgrid-config # TODO: Rename this 
+    cluster: test-infra-trusted # TODO: You may need to specify a cluster, depending on your Prow config
```
Comment (michelle192837) [inline testgrid/cmd/transfigure/README.md]: Should we not default to test-infra-trusted?

Comment (fejta) [reply testgrid/cmd/transfigure/README.md]: Do people create the job on k8s prow or other prow?

Comment (chases2) [reply testgrid/cmd/transfigure/README.md]: Other Prows. They may not have or use a trusted cluster, or may have a different alias for their cluster.

**Comment by chases2** [comment]
/hold cancel
Update to add directions to config.md regarding Transfigure

**Pull request merged**